### PR TITLE
Security consideration: more awareness about cross-origin sharing

### PR DESCRIPTION
**Pull request by ianbjacobs**
Based on:
https://docs.google.com/document/d/1i9ntZ1lEGyrzCAj1FnR9837u_rBU6vNoG0OZpiILKy0/edit

Implementation commitment:

 * [ ] Safari (link to issue)
 * [ ] Chrome (link to issue)
 * [ ] Firefox (link to issue)
 * [ ] Edge (public signal)

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/payment-handler/pull/366.html" title="Last updated on Apr 22, 2020, 5:52 PM UTC (7661ae7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/payment-handler/366/59f9268...7661ae7.html" title="Last updated on Apr 22, 2020, 5:52 PM UTC (7661ae7)">Diff</a>

### CODE REVIEW

**Comment by marcoscaceres** [comment]
https://github.com/w3c/payment-handler/pull/367 for tidy.

File: index.html [hunk 1]
```diff
@@ -2445,6 +2445,22 @@ <h2>
...
+          User Awareness about Sharing Data Cross-Origin
+        </h2>
+        <ul>
+          <li>By design, a payment handler from one origin shares data with
+          another origin (e.g., the merchant site).
+          </li>
+          <li>It is important that user agents make clear to users the origin
+          of a payment handler.
+          </li>
+          <li>User agents should help users understand that they are sharing
```
Comment (marcoscaceres) [inline index.html:2459]: This should probably state why it's important.

Comment (ianbjacobs) [reply index.html:2459]: Added rationale: to mitigate phishing attacks

Comment (danyao) [reply index.html:2459]: For the last point, maybe also suggest that user agents should help users understand what data they are sharing across origin?

Comment (ianbjacobs) [reply index.html:2459]: Hi @danyao, 

What level of detail do you think is practical? Some examples (verbiage illustrative only):

- Level 0: You are sharing personal information with <origin> for this transaction.
- Level 1: You are sharing address and contact information with <origin> for this transaction.
- Level 2: You are sharing the following with <origin>:
             <specific address>
             <specific contact information>

Ian

Comment (danyao) [reply index.html:2459]: I think it should be up to the browser vendor to decide which level, but the more details the better IMHO.

Comment (ianbjacobs) [reply index.html:2459]: To not overconstrain browsers I have added "and ideally what information they are sharing."

Comment (danyao) [reply index.html:2459]: Looks good. Thanks @ianbjacobs !

**Review by danyao — APPROVED** [review]

**Review by marcoscaceres — APPROVED** [review]
seems ok to me.

**Pull request merged**